docs/style.css

```diff
@@ -0,0 +1,236 @@
+#logo {
```

Comment:
If you want to you can replace some colors, or even move the colors to a specific CSS style so they are easy to change. So it can look different from the FOG engine one. But that's just a nit.